tests/python/frontend/pytorch/test_forward_refactor.py

```diff
@@ -0,0 +1,813 @@
+    verify_model(Multiply2().float().eval(), input_data=input_data)
+    verify_model(Multiply3().float().eval(), input_data=input_data)
+    verify_model(Multiply4().float().eval(), input_data=input_data)
+    verify_model(Multiply5().float().eval(), input_data=input_data)
+
+def test_forward_unsqueeze():
+    input_shape = [1, 3, 224, 224]
+
+    class Unsqueeze1(Module):
+        def forward(self, *args):
+            return args[0].unsqueeze(2)
+
+    input_data = torch.rand(input_shape).float()
+    verify_model(Unsqueeze1().float().eval(), input_data=input_data)
+
+def test_forward_concatenate():
+    input_shape = [1, 3, 224, 224]
+
+    class Concatenate1(Module):
+        def forward(self, *args):
+            return torch.cat([args[0][:, 0].unsqueeze(1), args[0][:, 1].unsqueeze(1)], 1)
+
+    class Concatenate2(Module):
+        def forward(self, *args):
+            a = (args[0][:, :, 0] + 2) * 7
+            b = (args[0][:, :, 1] + 3) * 11
+            c = (args[0][:, :, 2] + 5) * 13
+            return torch.cat([t.unsqueeze(2) for t in [a, b, c]], 2)
+
+    input_data = torch.rand(input_shape).float()
+    verify_model(Concatenate1().float().eval(), input_data=input_data)
+    verify_model(Concatenate2().float().eval(), input_data=input_data)
+
+def test_forward_relu():
+    input_shape = [1, 3, 224, 224]
+
+    class ReLU1(Module):
+        def forward(self, *args):
+            return torch.nn.ReLU()(args[0])
+
+    input_data = torch.rand(input_shape).float()
+    verify_model(ReLU1().float().eval(), input_data=input_data)
+
+def test_forward_adaptiveavgpool1():
+    input_shape = [1, 3, 224, 224]
+
+    class AdaptiveAvgPool2D1(Module):
+        def forward(self, *args):
+            return torch.nn.AdaptiveAvgPool2d([1, 1])(args[0])
+
+    input_data = torch.rand(input_shape).float()
+    verify_model(AdaptiveAvgPool2D1().float().eval(), input_data=input_data)
+
+def test_forward_adaptiveavgpool2():
+    input_shape = [1, 3, 224, 224]
+
+    class AdaptiveAvgPool2D2(Module):
+        def forward(self, *args):
+            return torch.nn.AdaptiveAvgPool2d([100, 100])(args[0])
+
+    input_data = torch.rand(input_shape).float()
+    verify_model(AdaptiveAvgPool2D2().float().eval(), input_data=input_data)
+
+def test_forward_adaptiveavgpool3():
+    input_shape = [1, 3, 224, 224]
+
+    class AdaptiveAvgPool2D3(Module):
+        def forward(self, *args):
+            return torch.nn.AdaptiveAvgPool2d([224, 224])(args[0])
+
+    input_data = torch.rand(input_shape).float()
+    verify_model(AdaptiveAvgPool2D3().float().eval(), input_data=input_data)
+
+def test_forward_maxpool1():
+    input_shape = [1, 3, 224, 224]
+
+    class MaxPool2D1(Module):
+        def forward(self, *args):
+            return torch.nn.MaxPool2d(kernel_size=[1, 1])(args[0])
+
+    input_data = torch.rand(input_shape).float()
+    verify_model(MaxPool2D1().float().eval(), input_data=input_data)
+
+def test_forward_maxpool2():
```

Comment:
Looks like you forgot to merge maxpool1, 2, and 3 into a single test as you did for other cases.